arrow-fx-android/build.gradle

```diff
@@ -0,0 +1,54 @@
+buildscript {
+    repositories {
+        google()
+    }
+    dependencies {
+        classpath("com.android.tools.build:gradle:3.6.0")
+    }
+}
+
+plugins {
+    id "maven-publish"
+    id "base"
+    id "net.rdrei.android.buildtimetracker"
+    id "org.jetbrains.dokka"
+    id "org.jlleitschuh.gradle.ktlint"
+    id "ru.vyarus.animalsniffer"
+}
+
+repositories {
+    google()
+    jcenter()
+}
+
+//apply from: "$SUBPROJECT_CONF"
+apply from: "$DOC_CONF"
+//apply from: "$PUBLISH_CONF"
```

Comment:
and this one too?

Reply:
Yes, both are part of the leftover point in the description: `Enable publish and submodule config on the module (currently failing)`